ioc/VERSION

```diff
@@ -0,0 +1 @@
+0.4.1 2019/01/05
```

Comment:
should maybe be auto-generated on release

Reply:
Cutting releases is a manual process for now.